pkg/armhelpers/interfaces.go

```diff
@@ -26,6 +26,24 @@ type VirtualMachineListResultPage interface {
 	Values() []compute.VirtualMachine
 }
 
```

Comment:
the interface for AKSEngineClient did not add the VMSS func, so they we couldn't fake them.